worker/mozillaGradingWorker/ciphersuites.go

```diff
@@ -0,0 +1,1295 @@
+package mozillaGradingWorker
+
+//OpenSSLCiphersuites contains all the ciphersuites recognised by OpenSSL
+var OpenSSLCiphersuites = `{
```

Comment:
I'm always concerned about maintainability when we define constants like these. I would prefer if we had a programmatic way to generate this json file, which turns out to be pretty easy with a few lines of bash.
```bash
#!/usr/bin/env bash
echo "{"
ctr=0
./cipherscan/openssl ciphers -V | while read code ign name proto kx au enc mac
do
    if [ $ctr -gt 0 ]; then echo ','; fi
    echo -n "\"$name\":{
    \"proto\": \"$proto\",
    \"kx\": \"$(echo $kx|cut -d '=' -f 2)\",
    \"au\": \"$(echo $au|cut -d '=' -f 2)\",
    \"mac\": \"$(echo $mac|cut -d '=' -f 2)\",
    \"encryption\": {
        \"cipher\": \"$(echo $enc | grep -Po 'Enc=\K[a-zA-Z0-9]+')\",
        \"key\": $(echo $enc | grep -Po 'Enc=[a-zA-Z0-9]+\(\K[0-9]+')
    }
  }"
    ctr=1
done
echo "
}"
```
I'm thinking that invoking a go generate function on build to create this file would be better than managing it by hand.